smoke-test/tests/cypress/cypress/e2e/mutations/add_users.js

```diff
@@ -1,47 +1,100 @@
-const tryToSignUp = () => {
-    let number = Math.floor(Math.random() * 100000);
-    let name = `Example Name ${number}`;
-    cy.enterTextInTestId("email", `example${number}@example.com`);
-    cy.enterTextInTestId("name", name);
-    cy.enterTextInTestId("password", "Example password");
-    cy.enterTextInTestId("confirmPassword", "Example password");
-
-    cy.mouseover("#title").click();
-    cy.waitTextVisible("Other").click();
+const test_id = Math.floor(Math.random() * 100000);
```

Comment:
If we take this out of the test and for some reason the cypress test is re-run would the test id change or not? This was inside the function to ensure that the test ID changes in case of re-runs otherwise test will fail due to user already existing in case of a re-run due to some part failing.

Reply:
When We resetting password we have required for which email we have to reset password for this scenario i have move email and testid outside to reuse this for reset password .